File: src/backend/odin/common/common-api/src/main/java/cn/code/odin/common/api/anno/MicroService.java
package cn.code.odin.common.api.anno;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;


@SpringBootApplication(scanBasePackages = "cn.code.odin")
@EnableDiscoveryClient
@EnableCircuitBreaker
public @interface MicroService {
}

File: src/backend/zeus/common/common-web/common-web-mvc/src/main/kotlin/cn/code/zeus/common/web/anno/MicroService.kt
package cn.code.zeus.common.web.anno

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.cloud.openfeign.EnableFeignClients

@SpringBootApplication(scanBasePackages = ["cn.code.zeus"])
@EnableDiscoveryClient
@EnableFeignClients(basePackages = ["cn.code.zeus"])
annotation class MicroService